packages/components/package.json

```diff
@@ -20,29 +20,19 @@
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "types": "dist/index.d.ts",
-  "exports": {
```

Comment:
Deleting this for now, we can reintroduce this when `Icon` is rebuilt.
https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3038118367/Solution+Preview+SVG+Icons